### PR TITLE
Fix formatting issue

### PR DESCRIPTION
**Pull request by 8sd**

### CODE REVIEW

**Comment by code-asher** [comment]
Funnily enough, the doc is actually correct as it currently stands. We
need to add the comma here as well: https://github.com/cdr/code-server/blob/db39eacfa1fa15b3f4aaa44e6469f71d58b4d07b/packages/server/src/cli.ts#L31

File: doc/self-hosted/index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Options:
   --cert <value>
   --cert-key <value>
   -e, --extensions-dir <dir>  Set the root path for extensions.
-  -d --user-data-dir <dir>    Specifies the directory that user data is kept in, useful when running as root.
+  -d, --user-data-dir <dir>    Specifies the directory that user data is kept in, useful when running as root.
```
Comment (lsmoura) [inline doc/self-hosted/index.md]: You have one extra space
```suggestion
  -d, --user-data-dir <dir>   Specifies the directory that user data is kept in, useful when running as root.
```

**Comment by 8sd** [comment]
@lsmoura thanks for the hint :)
@code-asher Ithanks, I didn't know there was this second help

**Comment by code-asher** [comment]
@8sd our final check requires the commits to be signed, once that's done we'll merge this in!

**Comment by 8sd** [comment]
The commits are now verified

**Pull request merged**